easybuild/easyconfigs/l/libXdmcp/libXdmcp-1.1.2-foss-2016a.eb

```diff
@@ -15,8 +15,10 @@ sources = [SOURCE_TAR_GZ]
 source_urls = ['http://xorg.freedesktop.org/archive/individual/lib/']
 
 builddependencies = [
+    ('pkg-config', '0.29')
```

Comment:
@jrbosch I managed to introduce a syntax error here, leading to this error:

```
TypeError: 'tuple' object is not callable
```

The problem is that there's a comma (`,`) missing at the end of this line, so please change this to:

```python
    ('pkg-config', '0.29'),
```